Dockerfile

```diff
@@ -2,19 +2,18 @@ FROM ubuntu:18.04
 
 RUN apt update
 RUN DEBIAN_FRONTEND=noninteractive apt install -yq --no-install-recommends php php-json php-curl php-ctype php-iconv php-pdo php-mysql php-dom php-zip composer apache2 libapache2-mod-php
-COPY . /var/www/rolendar
+COPY entrypoint.sh /opt/entrypoint.sh
+
+RUN mkdir /var/www/rolendar
+COPY larp.conf /var/www/rolendar/larp.conf
```

Comment:
@daria-sieroshtan 
Later you do
`RUN cp /var/www/rolendar/larp.conf /etc/apache2/sites-available`
Please find the way to simplify this :)